StreamChatClient_v3/WebSocketClient/Events/ReactionEvents_Tests.swift

```diff
@@ -0,0 +1,46 @@
+//
+// Copyright © 2020 Stream.io Inc. All rights reserved.
+//
+
+@testable import StreamChatClient_v3
+import XCTest
+
+final class ReactionEvents_Tests: XCTestCase {
+    let newJSON = XCTestCase.mockData(fromFile: "ReactionNew")
+    let updatedJSON = XCTestCase.mockData(fromFile: "ReactionUpdated")
+    let deletedJSON = XCTestCase.mockData(fromFile: "ReactionDeleted")
+    
+    let eventDecoder = EventDecoder<DefaultDataTypes>()
+    let userId = "broken-waterfall-5"
+    let cid = ChannelId(type: .messaging, id: "general")
+    let messageId = "0e042a9c-d648-4a28-8ed6-dbdb2b7b4779"
```

Comment:
Are these the data from the payload? If yes, it should be mentioned somewhere.

Reply:
It's just a value from JSON to check that it was parsed correctly.

Reply:
Can you please add a comment explaining this? It's not obvious.